app/src/main/res/values/strings.xml

```diff
@@ -40,4 +40,16 @@
     <string name="title_history">History</string>
     <string name="title_insights">Insights</string>
     <string name="title_profile">Profile</string>
+
+    <string name="tracking_tab_title">Tracking</string>
+    <string name="measuring_tab_title">Measuring</string>
+
+    <string name="study_burst_progress_bar_day">Day</string>
+    <string name="study_burst_progress_bar_title">Study Burst</string>
+    <string name="measuring_center_label">WALK AND BALANCE</string>
```

Comment:
I like to define these types of strings in title case and then use android:textAllCaps="true"